demos/toolbar/default-flexible-toolbar.html

```diff
@@ -55,6 +55,7 @@
       <span>Flexible Expansion Ratio: <span id="ratio">0</span></span>
     </footer>
 
+    <script src="/assets/common.js" async></script>
```

Comment:
Nit: Can we move this line below `material-components-web.js`? Ditto for other files.

Both for consistency with other demo pages, and because it feels more readable (IMO) to keep the order of imports as:

1. MDC Web library file(s)
2. Demo-specific files

Reply:
Oh, thanks for pointing that out. Somehow didn't realize that was the order we were doing it elsewhere.